test/functional/test-cid.js

```diff
@@ -596,8 +613,7 @@ describe('cid', () => {
         cookieName: 'cookie_name',
       }, hasConsent).then(c => {
         expect(c).to.exist;
-        expect(c).to
-            .equal('amp-sha384([1,2,3,0,0,0,0,0,0,0,0,0,0,0,0,15])');
+        expect(c).to.equal('amp-AQIDAAAAAAAAAAAAAAAADw');
```

Comment:
Hope we don't sometimes eliminate the trailing padding :)

Could we lift the length restriction?
You are aware that it won't always start with `amp-`, right?